app/models/event.rb

```diff
@@ -1,9 +1,14 @@
 # app/models/event.rb
 class Event < ApplicationRecord
+  NOTIFICATION_PERIOD = 1.day
```

Comment:
Totally. We are adding the Figaro gem in the oauth work that I am doing. Once that is configured I think that's a great idea.